src/components/tooltip-popup/styles.scss

```diff
@@ -13,41 +15,49 @@
 
   &__content {
     animation: fadein 700ms ease-out forwards;
+    margin-left: 8px;
 
     display: flex;
     align-items: center;
-    padding: 16px;
+    padding: 8px;
     max-width: 32rem;
     height: 24px;
-    background-color: theme.$color-primary-3;
     border-radius: 8px;
-    font-weight: 600;
-    line-height: 24px;
+    font-weight: 400;
+    line-height: 20px;
+    font-size: 14px;
 
-    /* Dark/Greyscale/Grey-12 */
-    color: theme.$color-greyscale-12;
+    @include glass-shadow-and-blur;
+    @include flat-thick;
+    backdrop-filter: blur(32px);
```

Comment:
Do we need this backdrop-filter if its set in `@include glass-shadow-and-blur;` ? Looks like duplication?

Reply:
actually `flat-thick` has a backdrop-filter of `64px` set, i added this here to override..